### DISCUSSION TITLE
Making Translation More Deterministic by Adjusting Parameters

### DISCUSSION THREAD

**Discussion by refracta** [3ru/gpt-translate#61] [Ideas]
Hello, thank you for creating such a great Action. I have incorporated it into my project and am using it well.

However, I noticed that whenever I modify the README.md, the translation uses slightly different expressions for paragraphs that I didn't change. Wouldn't adjusting parameters like `temperature`, `top_p`, `frequency_penalty`, and `presence_penalty` make the translation more deterministic?

I am attaching a few examples of my use cases for your reference.

COMMIT1: https://github.com/refracta/dcss-server/commit/107f3d5a8b7ae12e3ff86c405f44d8ed2608efa1
GPT-COMMIT1: https://github.com/refracta/dcss-server/commit/f6c3458caa0db55826a8282fef8af1b598b30274

COMMIT2: https://github.com/refracta/dcss-server/commit/52a817a7b068c5506159af0e89076d349a98d734 
GPT-COMMIT2: https://github.com/refracta/dcss-server/commit/34acea63f6b509d756a7e20571c5c560aece0dd8

**Comment by 3ru** [comment]
I've made it possible to specify additional parameters such as `temperature`, `top_p`, `frequency_penalty`, and `presence_penalty `in version [`v1.1.12`](https://github.com/3ru/gpt-translate/releases/tag/v1.1.12). 
These adjustments should help make the translations more deterministic and align more closely with your expectations!

**Reply by refracta** [reply]
Thank you very much for adding the feature.
I can't test it right away, but when I have time and find suitable presets for the parameters, I will post the parameter values here.
It would be great if others who see this discussion and are interested would share their own presets.
